alphadia/plexscoring.py

```diff
@@ -1895,6 +1895,9 @@ def __call__(
         )
         thread_count = 1 if debug else thread_count
 
+        if debug:
```

Comment:
could you gather this and the other two `if debug`s in l. 1893 and 1896 in one block?